packages/stable/src/types.ts

```diff
@@ -2005,6 +2005,25 @@ export interface RevisionCameraProperties {
   position?: Tuple3<number>;
 }
 
+export type RevisionMetadataUpdate =
+  | RevisionMetadataUpdateSet
+  | RevisionMetadataUpdateAddRemove;
+
+export interface RevisionMetadataUpdateSet {
+  set: { [key: string]: string };
+}
+
+export interface RevisionMetadataUpdateAddRemove {
+  /**
+   * Key/value pairs to add
+   */
+  add?: { [key: string]: string };
```

Comment:
@haakonflatval-cognite Maybe it is better if you create new type for key value pairs and reuse it. 

Ex:
```JS
export interface KeyValuePair {
[key: string]: string;
}

export interface RevisionMetadataUpdateAddRemove {
add: KeyValuePair;
}
```

Also I see that the metadata is optional in `SecurityCategory`. Do you need  `add` and `remove` here to be optional also?

Reply:
I've added RevisionMetadata as its own type